src/transforms/metric_to_log.rs

```diff
@@ -50,6 +50,14 @@ pub struct MetricToLogConfig {
     #[serde(default)]
     #[configurable(metadata(docs::hidden))]
     pub log_namespace: Option<bool>,
+
+    /// Should this transform encode tags using the enhanced encoding of [the `native_json`
+    /// codec][vector_native_json]?
+    ///
+    /// If set to `false`, tags will always be encoded as single string values using the last value
+    /// assigned to the tag.
+    #[serde(default = "crate::serde::default_false")]
+    pub enhanced_tags: bool,
```

Comment:
```suggestion
    /// If set to `false`, tags will always be encoded as single string values using the last value
    /// assigned to the tag.
    #[serde(default = "crate::serde::default_false")]
    pub enhanced_tags: bool,
```

Have we documented anything about "enhanced tags" to link to? Also maybe this should `(docs::hidden)` until we're ready to fully release?

Reply:
I'm going to reword it slightly but keep the longer explanation because the suggestion does not describe what the component will do if it is set to `true`.

We don't have anything to link to yet. It is on my task list.

This will be part of the next release (too much to revert to avoid it), so I see no reason to hide docs now.

Reply:
Yeah, I was thinking we could link to that larger article - probably on the metrics data model page - rather than filling the text here. We did something similar for the description on buffers.